templates/base.html

```diff
@@ -145,8 +143,6 @@
             <ul class="footer-items">
                 <li><a href="{% url 'privacy' %}">Privacy</a></li>
                 <li><a href="{% url 'terms' %}">Terms of Use</a></li>
-                <li><a href="{% url 'resources' %}">Resources</a></li>
```

Comment:
I eventually want to update them to be more current. So they are nice to have as a reference for now. We can delete them and I can just re-create them later if you think that would be better than having them hidden.